chore(docs): update Wistia access content (#14616)
newrelic/docs-website: src/content/docs/style-guide/images/embed-videos.mdx

Besides the docs site, Wistia hosts videos for [learn.newrelic.com](https://learn.newrelic.com/). Wistia is a good place to host videos you create as a writer, as well as videos we receive from other teams.

All writers should have admin-level access to Wistia so that they can upload, manage, and track the performance of videos. If an internal contributor wants to add a video, our team will upload and manage it on their behalf. We do not want to provide contributor access to Wistia to those outside of our team.

To upload videos to Wistia:

> 1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue with SSO**.

Reviewer: @ally-sassman I think we need to be more specific:
1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), and click **Login**.
2. Enter your work email, and then click **SSO**.
3. In **Wistia URL**, enter `newrelic` and then click **Continue with SSO**

Author: @rhetoric101 With your instructions above, I don't see a **Login** button. Instead, when I click that link, I'm taken directly to step 3 where I'm entering `newreli`.

2. Click **Content library > Content** and click **docs.newrelic.com Videos**.

> 3. Click **Add > Upload**. If you don't see this option, you can request collaborator permissions from #help-documentation.

Reviewer: Since we're writing this for internal writers, maybe we don't need to mention the non-happy path. Internal writers should be able to ask for help.

Author: Agreed, I'll remove this line

> 3. Click **Add > Upload**. If you don't see this option, you can request collaborator permissions from #help-documentation.

Reviewer: @ally-sassman maybe this is the place to mention (in general terms), how to get large videos from contributors. How about adding a comment like this: "If the contributor's video is large, you can have them upload it to our Google team drive" (without being specific about team drive name).

> 3. Click **Add > Upload**. If you don't see this option, you can request collaborator permissions from #help-documentation.

Author: ```suggestion
3. Click **Add > Upload**.
```

To use Wistia to host videos:

> 1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue to SSO**.

Reviewer: ```suggestion
1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue with SSO**.
```

> 2. Click **Content library > Content** and click **docs.newrelic.com Videos**.

Reviewer: @ally-sassman I think we might still need to tell users to click through the pages until they find **docs.newrelic.com Videos**--unless there's a faster way to find it.

Author: @rhetoric101 I don't image users see a ton of options. I only see this one project. Are you thinking users see more projects if they're involved with other teams?

> 1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue with SSO**.
> 2. Click **Content library > Content** and click **docs.newrelic.com Videos**.
> 3. Click **Add > Upload**. If you don't have access to add videos, request collaborator permissions from #help-documentation.
> 4. Click **Embed &#38; Share**.
> 5. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:

Author: ```suggestion
1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue with SSO**.
2. Click **Content library > Content** and click **docs.newrelic.com Videos**.
3. Click **Add > Upload**. If you don't have access, request collaborator permissions from #help-documentation.
4. Click **Embed &#38; Share**.
5. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:
```

To use Wistia to host videos:

1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue to SSO**.
2. Click **Content library > Content** and click **docs.newrelic.com Videos**.

> 4. Click **Add > Upload**.
> 5. Click **Embed &#38; Share**.
> 6. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:

Author: ```suggestion
3. Click **Add > Upload**. If you don't have access to add videos, request collaborator permissions from #help-documentation.
4. Click **Embed &#38; Share**.
5. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:
```
